.github/workflows/publish-coverage.yml

```
@@ -37,54 +29,80 @@ jobs:
       actions: read
       contents: read
+      pull-requests: read
     steps:
       - name: Configure git user
         run: |
           git config --global user.email "ci@example.com"
           git config --global user.name "CI"
       - name: Create GitHub App token
         id: app-token
-        uses: actions/create-github-app-token@v1
+        uses: actions/create-github-app-token@v3
```

actionlint (1.7.12)

[error] missing input "app-id" which is required by action "actions/create-github-app-token@v3". all required inputs are "app-id", "private-key"

(action)